client/src/main/java/tds/assessment/Assessment.java

```diff
@@ -12,9 +12,18 @@
     private String selectionAlgorithm;
     private float startAbility;
     private String subject;
+    private List<ItemConstraint> itemConstraints;
     private List<Segment> segments = new ArrayList<>();
 
-    private Assessment() {}
+    public Assessment() {}
```

Comment:
If we leave this as is the public constructor can go away since Objects all have an empty constructor.

General question about all mutable objects in this project, do we want to have everything to have setters?  Or should the constructor instead take in the minimal amount of values that would make for a valid object.  I'd use the DB (non null fields) as a starting place for those minimal properties.

We can discuss more in tomorrow's parking lot.